examples/SLAM/README.md

```diff
@@ -0,0 +1,20 @@
+# SLAM (World tracking) camera demo
+![Preview](previews/SLAM-example.webp?raw=true "Dynamically generated mesh")
+Runs an AR session using WebXR Device AP where available with a fallback to 8th-wall implementation.
+
+To successfully run in 
+- make sure `API_TOKEN_XR8` is defined in the index.html file
+```
+<script>
+  const API_TOKEN_XR8 = "your 8thwall project API token";
+</script>
+```
+
+- Make sure const `WEBXR_REQUIRED_FEATURES` and `WEBXR_OPTIONAL_FEATURES` are defined in the index.html file
+```
+<script>
+      const WEBXR_REQUIRED_FEATURES = [{{ webxrRequiredFeatures }}];
+      const WEBXR_OPTIONAL_FEATURES = [{{ webxrOptionalFeatures }}];
+</script>
+```
+- Make sure the app is running on HTTPS.
```

Comment:
This should probably be red, blinking and font size 30... Possibly there's some other "note" or "warning" that we could use? Would probably make sense to have as the first element.

Reply:
Maybe HTTPS should be a requirement for the whole wonderlandengine?
Even without 8th Wall, native webXR requires HTTPS.

Reply:
Not on localhost, though, you may use `http://localhost` for development 😌

Reply:
yes, but even on localhost and port forwarded android, 8th Wall and webXR API requires https. I believe same for immersive-vr.
So basically http is only if you're creating a 3D game without AR or VR.